pkgs/development/python-modules/pygraphviz/default.nix

```diff
@@ -36,9 +36,13 @@ buildPythonPackage rec {
   checkInputs = [ pytest ];
 
   checkPhase = ''
+    runHook preCheck
     pytest --pyargs pygraphviz
+    runHook postCheck
```

Comment:
This could probably use `pytestCheckHook` couldn't it?

Reply:
I tried `pytestFlagsArray = [ "--pyargs" "pygraphviz" ]` but it didn't work.